src/settings/components/SettingBlocks.tsx

```diff
@@ -310,10 +387,32 @@ const ModelSettingsComponent: React.FC<ModelSettingsComponentProps> = ({
     }
   };
 
+  const handleVerify = async () => {
+    if (newModel.name && newModel.provider) {
+      setIsVerifying(true);
+      await chainManager.chatModelManager
+        .verify({ ...newModel })
+        .then((res) => new Notice(res + " Model verification successful!"))
```

Comment:
We don't need `res` in this notice, just showing the success message is good enough.